### PR TITLE
JSCS config with violations fixed

### PR DESCRIPTION
**Pull request by mrjoelkemp**
Brings JSCS to grunt core to use the "grunt" preset.

Currently, the `maximumLineLength` rule is nullified because `test/` contains about 90 violations of the
preset's default of 120 chars per line.

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
Regarding the travis fail, should I just use jscs directly instead of integrating it into the Gruntfile?

File: .jscsrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+	"preset": "grunt",
```
Comment (sindresorhus) [inline .jscsrc]: fix indentation

File: Gruntfile.js [hunk 2]
```diff
@@ -35,6 +35,9 @@ module.exports = function(grunt) {
         node: true,
       }
     },
+    jscs: {
+      src: ['lib/**/*.js', 'internal-tasks/**/*.js', 'test/**/*.js', '!test/fixtures/**/*.js']
```
Comment (sindresorhus) [inline Gruntfile.js]: one string on each line

**Comment by mrjoelkemp** [comment]
Thanks for the review @sindresorhus. Fixed and rebased.

**Comment by sindresorhus** [comment]
Nice work @mrjoelkemp :)

**Pull request merged**